doc/CMakeLists.txt

```diff
@@ -104,6 +104,24 @@ set_target_properties(
     ADDITIONAL_CLEAN_FILES "${DOXY_OUT}"
 )
 
+#projects
+set(GEN_PROJECTS_REST_SCRIPT ${CMAKE_CURRENT_LIST_DIR}/_scripts/gen_west_projects_rest.py)
+
+#list(APPEND PROJECTS_ARGS --turbo-mode)
+
+add_custom_target(
+  projects
+  COMMAND ${CMAKE_COMMAND} -E env
+  ZEPHYR_BASE=${ZEPHYR_BASE}
+  ${PYTHON_EXECUTABLE} ${GEN_PROJECTS_REST_SCRIPT}
+    ${PROJECTS_ARGS}
+    ${DOCS_SRC_DIR}/projects
+  VERBATIM
+  USES_TERMINAL
+  COMMENT "Generating West Projects documentation..."
+)
+set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS ${GEN_PROJECTS_REST_SCRIPT})
```

Comment:
-1 on new rest generator scripts. This can be handled natively in Sphinx with an extension. Zephyr has a few examples, other useful ones are available here https://github.com/nrfconnect/sdk-nrf/tree/main/doc/_extensions for reference as well.

Reply:
first version posted, based on an extension from the sdk-nrf repo...